app/models/miq_schedule.rb

```diff
@@ -325,33 +298,6 @@ def validate_run_at
     end
   end
 
-  def validate_file_depot  # TODO: Do we need this if the validations are on the FileDepot classes?
-    if self.sched_action.kind_of?(Hash) && self.sched_action[:method] == "db_backup" && file_depot
-      errors.add(:file_depot, "is missing credentials") if !file_depot.uri.to_s.starts_with?("nfs") && file_depot.missing_credentials?
-      errors.add(:file_depot, "is missing uri") if file_depot.uri.blank?
-    end
-  end
-
-  def verify_file_depot(params)  # TODO: This logic belongs in the UI, not sure where
-    depot_class                = FileDepot.supported_protocols[params[:uri_prefix]]
-    depot                      = file_depot.class.name == depot_class ? file_depot : build_file_depot(:type => depot_class)
-    depot.name                 = params[:name]
-    uri                        = params[:uri]
-    api_port                   = params[:swift_api_port]
-    depot.aws_region           = params[:aws_region]
-    depot.openstack_region     = params[:openstack_region]
-    depot.keystone_api_version = params[:keystone_api_version]
-    depot.v3_domain_ident      = params[:v3_domain_ident]
-    depot.security_protocol    = params[:security_protocol]
-    depot.uri                  = api_port.blank? ? uri : depot.merged_uri(URI(uri), api_port)
-    if params[:save]
-      file_depot.save!
-      file_depot.update_authentication(:default => {:userid => params[:username], :password => params[:password]}) if (params[:username] || params[:password]) && depot.class.requires_credentials?
-    elsif depot.class.requires_credentials?
-      depot.verify_credentials(nil, params)
-    end
-  end
-
```

Comment:
ooh, so these verify/validate methods aren't needed anymore?  🎉  if true.

Reply:
`validate_file_depot` on the otherhad is specifically doing:

```ruby
self.sched_action[:method] == "db_backup"
```

So I believe that can be removed as it would always no-op going forward.

Reply:
hmm, I don't really follow what the callers are doing, so I can't tell if it's used anymore.

Reply:
`validate_file_depot` was just one of the validations up at the top.  Again, because it checks for `"db_backup"`, we should now never need it.

The other one is called specifically from the UI:

https://github.com/ManageIQ/manageiq-ui-classic/blob/5e3a5d7858224498f8fa09a01aaf268a4c00693c/app/controllers/ops_controller/settings/schedules.rb#L303
https://github.com/ManageIQ/manageiq-ui-classic/blob/ca60ee5665a1b17f2f12275d46a8433eef874d9b/app/controllers/application_controller.rb#L465-L468

Reply:
Again, `verify_file_depot` is still needed... if we still need the UI pieces, but `validate_file_depot` only checks for action types of `"db_backup"` which should eventually not be a thing.

(sorry... started typing this up yesterday and didn't finish... oops)